include/ddc/kernels/splines/spline_builder_batched.hpp

```diff
@@ -68,11 +76,7 @@ class SplineBuilderBatched
                 preconditionner_max_block_size)
         , m_vals_domain(vals_domain)
     {
-        static_assert(
-                BcXmin == BoundCond::PERIODIC && BcXmax == BoundCond::PERIODIC,
-                "Boundary conditions other than PERIODIC are not supported yet in "
-                "SpSplineBuilderBatched");
```

Comment:
It might be a good idea to keep a sanity check:
```
        static_assert(
                ((BcXmin == BoundCond::PERIODIC) == (BcXmax == BoundCond::PERIODIC)),
                "Incompatible boundary conditions in SpSplineBuilderBatched");
```